### PR TITLE
[DATA-850] Add Support for More Banks

### PR DESCRIPTION
**Pull request by ColeDrain**
- This pull request updates the Bank Enums with the Banks currently supported by Decide
- Also added a method to retrieve the list of supported banks

Banks Added: ALAT, ECOBANK, FCMB, FIDELITY,  GLOBUS, KEYSTONE, KUDA, PROVIDUS, POLARIS, STANBIC, STERLING, UBA, UNITY, UNION

### CODE REVIEW

**Comment by ColeDrain** [comment]
Attempts to close
[DATA-850](https://indicina1.atlassian.net/browse/DATA-850?atlOrigin=eyJpIjoiMDgzNGM1ZDdhNGIxNDcxMDkyNzBjOWUzMjIxNDUwOTMiLCJwIjoiaiJ9)
[DATA-851](https://indicina1.atlassian.net/browse/DATA-851?atlOrigin=eyJpIjoiNDMyYjYwZjgxZTM0NDExMmIwYTEyZjM4ZmFjNTM1YzUiLCJwIjoiaiJ9)

File: decide/models/enums.py [hunk 1]
```diff
@@ -9,15 +12,37 @@ class Currency(Enum):
...
+    POLARIS = "076"
+    STANBIC = "221"
+    STERLING = "232"
+    UBA = "033"
+    UNITY = "215"
+    UNION = "032"
     ZENITH = "057"
 
+    @classmethod
+    def get_bank_list(cls):
```
Comment (aderopoa) [inline decide/models/enums.py:38]: Shouldn't this be added to changelog file?

**Review by aderopoa — CHANGES_REQUESTED** [review]

File: decide/models/enums.py (hunk 1 above)
Comment (ColeDrain) [reply decide/models/enums.py:38]: @aderopoa thanks for your question
The Changelog is automatically generated(updated) with the commit messages, the commit messages already references that a new method was added, will also add to documentation. I hope my answer is on point?

Comment (ColeDrain) [reply decide/models/enums.py:38]: The auto generation happens after the PR is merged

**Review by maxi-indicina — APPROVED** [review]

**Review by aderopoa — APPROVED** [review]

**Pull request merged**